go/chat/boxer.go

```diff
@@ -82,20 +85,20 @@ func (b *Boxer) UnboxMessage(ctx context.Context, boxed chat1.MessageBoxed, fina
 		return chat1.MessageUnboxed{}, NewTransientUnboxingError(err)
 	}
 
-	var matchKey *keybase1.CryptKey
+	var encryptionKey *keybase1.CryptKey
 	for _, key := range keys.CryptKeys {
 		if key.KeyGeneration == boxed.KeyGeneration {
-			matchKey = &key
+			encryptionKey = &key
 			break
 		}
 	}
 
-	if matchKey == nil {
+	if encryptionKey == nil {
 		err := fmt.Errorf("no key found for generation %d", boxed.KeyGeneration)
 		return chat1.MessageUnboxed{}, NewTransientUnboxingError(err)
 	}
 
-	umwkr, ierr := b.unboxMessageWithKey(ctx, boxed, matchKey)
+	unboxed, ierr := b.unlock(ctx, boxed, encryptionKey)
```

Comment:
Can we use a different name than `unlock`? It conflicts with the normal usage in a Go program for concurrency.